source/material_model/grain_size.cc

```diff
@@ -25,6 +25,7 @@
 #include <aspect/heating_model/shear_heating.h>
 #include <aspect/material_model/rheology/visco_plastic.h>
 #include <aspect/utilities.h>
+#include <aspect/simulator.h>
```

Comment:
do you need the simulator include here? The signals header is not enough?

Reply:
Yes, it doesn't compile without it.